### DISCUSSION TITLE
More ModBusClient

### DISCUSSION THREAD

**Discussion by knotzchef** [WillCodeForCats/solaredge-modbus-multi#506] [General]
Hello,
with ne newest SE Software the Inverer can handle more than 3 Clients
in my case, now HA Plugin, Wallbox, SmartHomeGateway and Waterheater are running at the same time

-> on advice, your configuration side with the screenshot from the SetApp are from an old version, so we have to update the scrrenshots

my Topic is, how can i read the data from the second inverter which is connected via ModBus-2 over RS485 to the 1. Inverter
-> like it is shown on your page it is not possible.

**Comment by WillCodeForCats** [comment]
The screenshots are from my inverters. I don't have access to newer firmware versions to provide updates, so someone will contribute that for new versions or you will need to figure it out and report back what the changes should be.

**Comment by knotzchef** [comment]
HI,
i can do it for you..... i make some new pictures to update your configuration side....
more clients are working with the 3-phase inverter since 4.19.36..
